README.md

```diff
@@ -1,145 +1,132 @@
+
+Below is a list of general areas that I'm planning to work on, so if you'd like to help there ping me or file an issue (I am 'arschles' on the Gophers Slack):
+
+* New storage backends (probably disk next because it's the easiest on the TODO list :smile:)
+* Adding new commands to the CLI (I'll probably use [Cobra](https://github.com/spf13/cobra) to do it)
+* Bugfixes in the CLI
+* Adding tests
+
+The only thing we ask is that you follow the
+[Contributor Covenant](https://www.contributor-covenant.org/).
+
+# Resources:
+
+* ["Go and Versioning"](https://research.swtch.com/vgo) papers
+* [vgo wiki](https://github.com/golang/go/wiki/vgo)
+
+# Does it Work?
+
+Great question (especially for an alpha project)! The short answer is this:
+
+> The basic pieces are in place for a proxy, but the CLI and the server makes
+> it near-impossible to use this thing in the real world
+
+## Some Additional Details
+
+The basic API and storage system work, but the proxy is limited for real world use right now.
+
+First, it only stores modules in memory, so that's a major issue if you want to use it for anything
+real.
+
+Second, it doesn't hold any packages other than the ones you upload to it. A package proxy is pretty much only as useful as the packages it stores. You can work around that by declaring dependencies as `file:///` URLs if you want, but that defeats much of the purpose of this project.
+
+When athens has better storage drivers (at least persistent ones!), it'll be easier to load it up with modules (i.e. by running a background job to crawl your `GOPATH`). At that point, it'll be more practical to successfully run `vgo get` inside a less-trivial project.
+
+Finally, here's what the whole workflow looks like in the real world (spoiler alert: the CLI needs work). The setup:
+
+* First, I uploaded a basic module to the server using the CLI (see above) using the following command from the root of this repo:
+  `console ./athens ./testmodule arschles.com testmodule v1.0.0`
+* Then I created a new module with the following files in it:
+  * A single `go.mod` file with only the following line in it: `module "foo.bar/baz"`
+  * A `main.go` file with the following in it:
+
+```go
+package main
+    func main() {}
+```
+
+Finally, from the root of the new module, I ran `vgo get arschles.com/testmodule@v1.0.0` and got the following output:
+
+```console
+$ vgo get arschles.com/testmodule@v1.0.0
+vgo: downloading arschles.com/testmodule v1.0.0
+vgo: import "arschles.com/testmodule": zip for arschles.com/testmodule@v1.0. has unexpected file testmodule/.DS_Store
+```
+
+As you can see, the CLI uploaded a file to athens that's not `.go`, `go.mod`, or anything else that vgo, so at least the CLI needs some work (and the server needs some sanity checks too).
+
+You can get around all of this by manually zipping up your code and uploading it with `curl` or similar, but like I said, that's super impractical. Yay alpha software!
```

Comment:
this whole "additional details" section is out of date in various ways, so we can probably remove it (although I do like the "yay alpha software!" bit - can we keep that somehow?? 😄)